wezterm-font/src/fcwrap.rs

```diff
@@ -125,7 +125,7 @@ impl<'a> CharSetRef<'a> {
                 for j in 0..32 {
                     if mask & (1 << j) != 0 {
                         let new_code_point = base_code_point + (j + i * 32) as u32;
-                        if new_code_point - 1 > code_point {
+                        if new_code_point > 0 && new_code_point - 1 > code_point {
```

Comment:
ah, this looks like it doesn't belong in this PR!

Reply:
I'll push this in as a separate commit (bd09b568ed31401a5b525aad40c5d3817e55e486); hopefully this one will rebase out without issue